debian/bionic/foreman/control

```diff
@@ -210,7 +210,7 @@ Package: foreman-service
 Architecture: all
 Section: web
 Priority: extra
-Depends: foreman (= ${binary:Version}), ${misc:Depends}
+Depends: foreman (= ${binary:Version}), puma-status, ${misc:Depends}
```

Comment:
where is this package coming from? I don't see it in Debian or our repos?

Reply:
Honestly, I don't know what determines when a package comes from one place or the other given we use bundle install for dependencies?

Reply:
We bundle things that are in bundler.d, not from the Debian Depends.